src/main/scala/scorex/core/app/Application.scala

```diff
@@ -100,20 +98,8 @@ trait Application extends ScorexLogging {
     Runtime.getRuntime.addShutdownHook(new Thread() {
       override def run() {
         log.error("Unexpected shutdown")
-        stopAll()
+        ErgoApp.shutdownSystem()
       }
     })
   }
-
-  def stopAll(): Unit = synchronized {
-    log.info("Stopping network services")
-    upnpGateway.foreach(_.deletePort(scorexSettings.network.bindAddress.getPort))
```

Comment:
this line is lost, why not needed?

Reply:
https://github.com/ergoplatform/ergo/blob/master/src/main/scala/org/ergoplatform/ErgoApp.scala#L158